zebra-rpc/src/methods/tests/snapshot.rs

```diff
@@ -943,12 +950,11 @@ pub async fn test_mining_rpcs<ReadState>(
 
     #[allow(clippy::unnecessary_struct_initialization)]
     let mining_conf = crate::config::mining::Config {
-        miner_address: Some(ZcashAddress::from_transparent_p2pkh(
+        miner_address: Some(ZcashAddress::from_transparent_p2sh(
```

Comment:
This is a fix for https://github.com/ZcashFoundation/zebra/pull/9574#discussion_r2136753083, and is not relevant to the issue this PR is intended to close. It also causes changes in snapshots that pertain to it. I included the change here since the original PR is merged.